autogen/token_count_utils.py

```diff
@@ -13,16 +13,19 @@ def get_max_token_limit(model="gpt-3.5-turbo-0613"):
         "gpt-3.5-turbo-0301": 4096,
         "gpt-3.5-turbo-0613": 4096,
         "gpt-3.5-turbo-instruct": 4096,
-        "gpt-3.5-turbo-16k": 16384,
+        "gpt-3.5-turbo-16k": 16385,
+        "gpt-3.5-turbo-1106": 16385,
```

Comment:
Also add `"gpt-3.5-turbo-16k-0613": 16385`?

what about remove model starts with `gpt-35-turbo`, and replace `gpt-35-turbo` with `gpt-3.5-turbo` at the beginning of the function?

Reply:
I assume the gpt-35-turbo models were there for common Azure deployment reasons? I don't know their origin.

Reply:
Thanks @afourney . gpt-35-turbo is for Azure OpenAI models. I think you missed adding `model = model.replace("gpt-35-turbo", "gpt-3.5-turbo")`.